wscript

```diff
@@ -253,6 +255,18 @@ def handle_configure_options(conf):
         conf.env.append_value('DEFINES', 'VERBOSE_LOGGING')
         print("Verbose logging enabled")
 
+    print(f"Log level: {conf.options.log_level}")
```

Comment:
nit: maybe just make the enum values match the ends of the defines so you can just uppercase them and not need a bunch if/elif

Reply:
done